examples/open_source_models/role_playing_with_open_source_model.py

```diff
@@ -87,4 +85,11 @@ def main(model_type=None) -> None:
 
 
 if __name__ == "__main__":
-    main(ModelType.LLAMA_2)
+    # Here :obj:`model_type` can be any of the supported open-source
+    # model types and :obj:`model_path` should be set corresponding to
+    # model type. 
+    main(
+        model_type=ModelType.LLAMA_2,
+        model_path="meta-llama/Llama-2-7b-chat-hf",
+        server_url="http://localhost:8000/v1",
+    )
```

Comment:
Great! Can we also put vicuna example params here but comment out? So people can try it out as well.

Reply:
Will do :)